backend/daemon/api/documents/v1alpha/documents.go

```diff
@@ -36,19 +36,21 @@ type Discoverer interface {
 
 // Server implements DocumentsServer gRPC API.
 type Server struct {
-	db    *sqlitex.Pool
-	vcsdb *sqlitevcs.DB
-	me    *future.ReadOnly[core.Identity]
-	disc  Discoverer
+	db      *sqlitex.Pool
+	vcsdb   *sqlitevcs.DB
+	me      *future.ReadOnly[core.Identity]
+	sitesDB map[string]siteInfo //TODO: remove when finished with the mockup
```

Comment:
@juligasa maybe guard this with a mutex, or for this simple use case maybe just use a [sync.Map](https://pkg.go.dev/sync#Map)? Even though it's weird to get concurrent requests here, I wouldn't discard that it could happen, due to the many layers of abstraction we have on the frontend (Vite dev server, react-query, grpc-codegen, react hooks, etc.).

Reply:
I thought about that, but since this `sitesDB` is going to be replaced with a proper database call, then concurrency will be handled in sqlite... @burdiyan